### PR TITLE
Update to 43.0

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

**Comment by A6GibKm** [comment]
bot, build

**Comment by A6GibKm** [comment]
bot, cancel 112052

**Comment by A6GibKm** [comment]
I can confirm that the app works.

File: metainfo-caption.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/data/org.gnome.Contacts.appdata.xml.in.in b/data/org.gnome.Contacts.appdata.xml.in.in
+index 98bdedfddf9eb8987fe8aac1adab9b738fde697d..6397e21dac686f40c522e0af09fc4aae13aa3939 100644
+--- a/data/org.gnome.Contacts.appdata.xml.in.in
++++ b/data/org.gnome.Contacts.appdata.xml.in.in
+@@ -19,7 +19,7 @@
+   </description>
+   <screenshots>
+     <screenshot type="default">
+-      <caption>Contacts with no contacts.</caption>
++      <caption>Contacts with no contacts</caption>
```
Comment (bilelmoussaoui) [inline metainfo-caption.patch:10]: Is this patch actually needed?

Comment (A6GibKm) [reply metainfo-caption.patch:10]: Yes appstream will have none of this. Patch already submitted upstream

**Pull request merged**